### ISSUE TITLE
Feature request: Support for month format MMM

### ISSUE THREAD

**Issue by marekhattas** [pushtell/react-bootstrap-date-picker#113]
Hi,, it will be nice if you can support also another date formats,,, e.g. month MMM,,
like https://momentjs.com/docs/#/displaying/

**Comment by jimmcslim** [comment]
I'd add that support for month-of-year selection (returning a date defaulting to the first of month in the year) would be good as well.

**Comment by vadym-kovalenko** [comment]
Yeah it would be great to have support for momentjs' date formats
